web/tests/notifications.test.js

```diff
@@ -367,7 +367,15 @@ test("basic_notifications", () => {
         }
     }
 
-    desktop_notifications.set_notification_api(StubNotification);
+    // TODO(evy): I'm getting this error:
+    // > You are overriding a non-function with a function.
+    // > This is almost certainly an error.
+    // 
+    // Do I need to go back to having a `set_notification_api` function?
+    // that can create some typing issues because NotificationAPI and
+    // StubNotification are different types. Maybe there are other, better,
+    // ways to resolve this. Asking here for advice :)
```

Comment:
Question here!

Reply:
We want to remove uses of rewire, not add them. Keep `set_notification_api`. The Node tests aren’t TypeScript and this is hardly the only type-unsafe stub in them.

Reply:
`set_notification_api` is in the main module though, would I just type it incorrectly as if it was receiving a proper NotificationAPI object?

Reply:
Right.